### PR TITLE
feat(go-mobile): add mobile npm package

### PR DESCRIPTION
**Pull request by sanderpick**
Adds `@textile/go-mobile` npm package to the `mobile` directory.

### CODE REVIEW

File: mobile/scripts/fetch_release.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+DIR="$(dirname "$(stat -f "$0")")"
+IOS_DIR="${DIR}/../ios"
+ANDROID_DIR="${DIR}/../android"
+
+# fetch iOS framework
+mkdir -p ${IOS_DIR}
+wget https://github.com/textileio/textile-go/releases/download/v0.1.4/textile-go_v0.1.4_ios-framework.tar.gz
```
Comment (asutula) [inline mobile/scripts/fetch_release.sh]: It's fine all these version numbers are hard coded for now?

Comment (asutula) [reply mobile/scripts/fetch_release.sh]: Seems like the version string could be passed as an arg from package.json to this script

Comment (sanderpick) [reply mobile/scripts/fetch_release.sh]: it's in a half-baked state now where this package version doesn't match the go release version (cause i had already published 0.1.4)... so, next release can clean all that up... ideally, in CI we'd bump both versions together, then auto-publish this... something like pass version to script, like you suggest. fine with it like this until next time around

**Comment by sanderpick** [comment]
Slack down... but good 2 go here.

**Comment by sanderpick** [comment]
`@textile/go-mobile@0.1.5-rc.7` is latest w/ curl

File: mobile/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "keywords": [
+    "textile",
+    "decentralized",
+    "mobile"
+  ],
+  "files": [
+    "scripts"
+  ],
+  "scripts": {
+    "postinstall": "./scripts/fetch_release.sh 0.1.5"
```
Comment (asutula) [inline mobile/package.json]: We should just use `$npm_package_version` here so you don't ever have to update that

Comment (sanderpick) [reply mobile/package.json]: oh nice, didnt know about that

Comment (asutula) [reply mobile/package.json]: Think it works, def test it haha

Comment (sanderpick) [reply mobile/package.json]: yep, works

**Pull request merged**